platforms/r3-corda-ent/configuration/roles/setup/zone/README.md

```diff
@@ -0,0 +1,54 @@
+## ROLE: `setup/zone`
```

Comment:
add copyright similar to other .md files

Reply:
fixed